proof/crefine/Move_C.thy

```diff
@@ -11,6 +11,11 @@ theory Move_C
 imports Refine.Refine
 begin
 
+lemma and_one_neq_simps[simp]:
+  "a AND 1 \<noteq> 0 \<longleftrightarrow> a AND 1 = 1"
+  "a AND 1 \<noteq> 1 \<longleftrightarrow> a AND 1 = 0"
+  by (clarsimp simp: and_one_eq)+
```

Comment:
This is a temporary location for this lemma for reviewing purposes, once https://github.com/seL4/l4v/pull/587 is merged I'm planning to move it to `lib/Word_Lib/More_Bit_Ring.thy`.